packages/snaps-cli/src/commands/watch/watch.ts

```diff
@@ -0,0 +1,66 @@
+import { isFile } from '@metamask/snaps-utils';
+
+import { ProcessedConfig, ProcessedWebpackConfig } from '../../config';
+import { CommandError } from '../../errors';
+import { executeSteps, info, Steps } from '../../utils';
+import { getServer } from '../../webpack';
+import { watch } from './implementation';
+
+type WatchOptions = {
+  /**
+   * The port to listen on.
+   */
+  port?: number;
+};
+
+type WatchContext = {
+  config: ProcessedWebpackConfig;
+  options: WatchOptions;
+};
+
+const steps: Steps<WatchContext> = [
+  {
+    name: 'Checking the input file.',
+    task: async ({ config }) => {
+      const { input } = config;
+
+      if (!(await isFile(input))) {
+        throw new CommandError(
+          `Input file not found: "${input}". Make sure that the "input" field in your snap config is correct.`,
+        );
+      }
+    },
+  },
+  {
+    name: 'Starting the development server.',
+    task: async ({ config, options, spinner }) => {
+      const server = getServer(config);
```

Comment:
Previously you could disable serving, do we want to keep supporting that?

Reply:
Sure, we could add a `server.enabled` option? Not sure when you would want to disable it, but it's trivial to add.

Reply:
Otherwise `mm-snap watch --serve=false` will not work anymore?